docs/EN_US/ContainerizedHPCC/ContainerizedMods/TroubleshootingHelmDeployments.xml

```diff
@@ -0,0 +1,516 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="TroubleshootingContainerizedDeployments">
+  <title>Troubleshooting Containerized Deployments</title>
+
+  <sect1 id="introductionTroubleshooting">
+    <title>Introduction</title>
+
+    <para>Helm is a powerful package manager for Kubernetes, simplifying the
+    deployment and management of complex applications. However, even with
+    Helm, deployment issues can arise. This chapter will guide you through
+    common troubleshooting steps for Helm deployments. Command-line tools,
+    such as <emphasis>kubectl</emphasis> and <emphasis>helm</emphasis> are
+    available for both local and cloud deployments.</para>
+  </sect1>
+
+  <sect1 id="UsefulHelmCommands">
+    <title>Useful Helm Commands</title>
+
+    <para>Here are some useful Helm commands for troubleshooting.</para>
+
+    <para><emphasis role="bold">List deployments</emphasis> using this command
+    in a terminal window:</para>
+
+    <para><programlisting>helm list</programlisting></para>
+
+    <para>This returns all installed Helm releases.</para>
+
+    <para>If you have multiple namespaces, use this command:</para>
+
+    <para><programlisting>helm list -A </programlisting></para>
+
+    <para>Returns all installed Helm releases across all namespaces.</para>
+
+    <para><emphasis role="bold">Get the status</emphasis> of a specific
+    release using this command in a terminal window:</para>
+
+    <para><programlisting>helm status &lt;release-name&gt;</programlisting></para>
+
+    <para>This returns the status of a specific release.</para>
+
+    <para><emphasis role="bold">Get the user supplied values</emphasis> for a
+    release using this command in a terminal window:</para>
+
+    <para><programlisting>helm get values &lt;release-name&gt;</programlisting></para>
+
+    <para>By effectively using these Helm commands, you can quickly identify
+    and resolve issues with your Helm deployments. Remember to consult the
+    official Helm documentation for more detailed information and specific use
+    cases.</para>
+  </sect1>
+
+  <sect1 id="checkthestatusofpods">
+    <title>Check the Status of Pods</title>
+
+    <para>Pods are the smallest deployable units of computing that can be
+    created and managed in Kubernetes. Checking the status of pods is a
+    fundamental step in troubleshooting Kubernetes deployments. By monitoring
+    pod status, you can quickly identify and address potential issues,
+    ensuring the health and performance of your applications.  </para>
+
+    <para>The HPCC Systems platform has a pod for each component of a system.</para>
```

Comment:
.. has 1 or more pod corresponding to each component of a deployed system. ?

(e.g. Thor has 2 static agents, and other dynamically launched pods for the manager + workers)